APMrover2/GCS_Mavlink.cpp

```diff
@@ -417,6 +417,8 @@ bool GCS_MAVLINK_Rover::try_send_message(enum ap_message id)
     case MSG_RANGEFINDER:
         CHECK_PAYLOAD_SIZE(RANGEFINDER);
         rover.send_rangefinder(chan);
+        CHECK_PAYLOAD_SIZE(DISTANCE_SENSOR);
```

Comment:
You can remove this check, since the next one will check for each message it tries to send.